http_logs/index.json

```diff
@@ -9,19 +9,52 @@
     "_source": {
       "enabled": {{ source_enabled | default(true) | tojson }}
     },
+    {%- if runtime_script_grok is defined %}
```

Comment:
Similarly I think we should limit the amount of files that contain conditionals. I suggest we have a separate `index-runtimefields.json` and apply it conditional in the central `track.json` where we anyway have some conditionals.

Reply:
I hadn't realized that was a thing. I'm happy to do that.